constraint/pkg/client/drivers/rego/schema/schema.go

```diff
@@ -49,22 +52,36 @@ func GetSource(code templates.Code) (*Source, error) {
 	if !ok {
 		return nil, ErrBadType
 	}
+
 	source := &Source{}
+
 	rego, found, err := unstructured.NestedString(v, "rego")
 	if err != nil {
 		return nil, fmt.Errorf("%w: while extracting Rego source", err)
 	}
 	if !found {
 		return nil, fmt.Errorf("%w: rego", ErrMissingField)
 	}
+
 	source.Rego = rego
 
+	version, found, err := unstructured.NestedString(v, "version")
+	if err != nil {
+		return nil, fmt.Errorf("%w: while extracting Rego version", err)
+	}
+	if !found {
+		version = "v0"
```

Comment:
is there a test that validates this

Reply:
Thanks for flagging, one added in fee23bf54352a13758430f9ebf22a7f68c7c33fd